conf/mesos-site.xml

```diff
@@ -53,6 +53,16 @@
   </property>
 
   <property>
+    <name>mesos.hdfs.mesosdns</name>
```

Comment:
No problem. Let's follow hadoop convention then. I get used to the terseness of Mesos/C++, so sometimes it's hard for me to context-switch back into the verboseness of Java/Hadoop. :)